src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,446 @@
+| TubingLength                        | RECOMMENDED           | [number][]    | The length of the blood tubing, from the subject to the detector in meters.                                                                                                                                                              |
+| DispersionConstant                  | RECOMMENDED           | [number][]    | External dispersion time constant resulting from tubing in default unit seconds.                                                                                                                                                                              |
+| Haematocrit                         | RECOMMENDED           | [number][]    | Measured haematocrit, meaning the volume of erythrocytes divided by the volume of whole blood.                                                                                                                                                                |
+| BloodDensity                        | RECOMMENDED           | [number][]    | Measured blood density. Unit of blood density should be in g/mL.                                                                                                                                                                                              |
+
+### Time
+
+If manual blood samples, for example, using a COBRA counter are available,
+then time information as detailed below should be added to the `*_recording-manual_blood.tsv`
+and the values below should be added to the `*_recording-manual_blood.json`.
+If blood samples using an autosampler, for example, using an Allogg autosampler are available,
+then time information as detailed below should be added to the `*_recording-autosampler_blood.tsv`
+and the values below should be added to the `*_recording-autosampler_blood.json`.
+
+| **Column name** | **Description**                                                              | **Units**                             |
+| --------------- | ---------------------------------------------------------------------------- | ------------------------------------- |
+| time            | Time in relation to `TimeZero` defined by the `*_pet.json`. (for example, 5) | Unit of time steps in seconds |
+
+### Radioactivity in plasma
+
+This section may be omitted if plasma measurements of radioactivity were not made.
+It contains information regarding sampled plasma data.
+If plasma measurements are available by manual sampling, they should be added to the `*_recording-blood_discrete.tsv`
+and the values below should be added to the `*_recording-blood_discrete.json`.
+If plasma measurements are available by an autosampler, they should be added to the `*_recording-blood_continous.tsv`
+and the values below should be added to the `*_recording-blood_continous.json`.
```

Comment:
Should this be `*_recording-manual_blood.tsv` and `*_recording-autosampler_blood.tsv` instead of `*_recording-blood_discrete.tsv` and `*_recording-blood_continuous.tsv`, respectively?

Reply:
It looks like this is addressed with https://github.com/bids-standard/bids-specification/pull/633#discussion_r563800522 and other proposed edits, so I will resolve this.